AUTHORS

```diff
@@ -9,3 +9,4 @@ Dennis Jen <djen@edx.org>
 Andy Armstrong <andya@edx.org>
 Alasdair Swan <aswan@edx.org>
 Alisan Tang <atang@edx.org>
+Brian Jacobel <bjacobel@edx.org>
```

Comment:
This doesn't affect Zappr, I'm just adding it here because I noticed I hadn't added myself yet.